Source/Core/Core/FreeLookManager.cpp

```diff
@@ -220,48 +229,50 @@ void FreeLookController::Update()
   const auto gyro_motion_quat =
       Common::Quaternion::RotateXYZ(gyro_motion_rad_velocity_converted * dt);
 
-  g_freelook_camera.Rotate(gyro_motion_quat);
+  auto* camera_controller = static_cast<CameraControllerInput*>(g_freelook_camera.GetController());
```

Comment:
I feel it might be better to have `UpdateInput` take a `CameraControllerInput*` as a parameter; that way, the cast would be in the same place as the `camera_controller->SupportsInput()` check.